### PR TITLE
Adding Cypress with a few starter tests

### PR DESCRIPTION
**Pull request by Roughnick**
# Author Section

## Description

Added Cypress for E2E testing. Since I am not sure of all the current features I just added some bases tests. Currently cypress is just setup for local testing. This will probably need a CI configuration if the project has progressed that far.

Also left in the Example tests just for review if others were not familiar with cypress. I was planning to remove before merging.

## PR Readiness Checklist

- [x] Verified all changes are related to feature described above (no scope creep)
- [x] Checked target and base branch (should be feature branch merging INTO `main`)
- [x] Added unit tests to cover modified / additional code
- [x] Added assignee

### CODE REVIEW

File: test/SimpleCraApp.E2ETests/cypress/integration/simplePage.spec.js
Comment (Wolven531) [inline test/SimpleCraApp.E2ETests/cypress/integration/simplePage.spec.js:22]: For consistency, let's keep a newline at the end of each file

```suggestion
})

```

File: test/SimpleCraApp.E2ETests/package.json
Comment (Wolven531) [inline test/SimpleCraApp.E2ETests/package.json:11]: Technically the upper package has an MIT license, so we should update this one to match

```suggestion
  "license": "MIT",
```

**Review by Wolven531 — CHANGES_REQUESTED** [review]
Couple of minor changes requested; also, I think you may be using a different version of NPM, which in turn causes your `npm i` command to update the project's package-lock; specifically, the project is using `lockfileVersion` of 2. I'm locally using Node v16.3.0 and NPM 7.15.1, so I'll checkout your branch and update this file to fix it

**Review by Wolven531 — APPROVED** [review]
Lgtm, 👍

**Comment by Roughnick** [comment]
> Lgtm, 👍

I guess you didn't want to wait for changes to be pushed up haha 😄

**Comment by Wolven531** [comment]
> > Lgtm, 👍
> 
> I guess you didn't want to wait for changes to be pushed up haha 😄

Normally I would, but for the initial commit stuff there was a bunch of tiny little tweaks that would have been annoying to detail, lol

Nice work on the initial test, though. I haven't yet hooked it up to run anywhere, but we can likely use these tests in a GitHub Action and instead of aiming at localhost, we'll aim at staging or something. This is a great first step toward that

**Comment by Wolven531** [comment]
Also, now that it's approved - feel free to merge at will. Right now, the only restriction is that at least one Code Owner must review and approve before merge. If you'd like to wait for @VincentLeighton 's review, that's fine as well. =]

**Review by VincentLeighton — APPROVED** [review]
Looks like gibberish to me but okay!

**Pull request merged**
